evalml/utils/gen_utils.py

```diff
@@ -490,3 +491,38 @@ def deprecate_arg(old_arg, new_arg, old_value, new_value):
         )
         value_to_use = old_value
     return value_to_use
+
+
+class suppress_stdout_stderr(object):
```

Comment:
Omg prophet 😓  I don't know how I feel about opening files in the os from a component. Will this work if two prophet regressors are running simultaneously? 

I agree that supressing the stan output is prefered but is there something else we can do?

Reply:
I removed this bit, it seems like it was PyStan that was logging all of that output and we're relying on Cmdstanpy